server/src/main/scala/org/http4s/server/staticcontent/ResourceService.scala

```diff
@@ -3,14 +3,14 @@ package server
 package staticcontent
 
 import cats.data.{Kleisli, OptionT}
-import cats.effect._
+import cats.effect.{Blocker, ContextShift, Sync}
 
 object ResourceService {
 
   /** [[org.http4s.server.staticcontent.ResourceService]] configuration
     *
     * @param basePath prefix of the path files will be served from
-    * @param blockingExecutionContext `ExecutionContext` to use when collecting content
+    * @param blocker `ExecutionContext` to use when collecting content
```

Comment:
Maybe update this to refer to `cats.effect.Blocker` while you're at it.

Reply:
Replaced "`ExecutionContext`" with "execution context" to be consistent with the other docblock.